extensions/amp-story/1.0/amp-story.css

```diff
@@ -1001,4 +1001,28 @@ amp-story-quiz {
   /* 224px = the width of a narrow phone (320px) - tap-safe margins (48px * 2) */
   min-width: 224px !important;
   max-width: 400px !important;
+  --accent-color: #005AF0;
+  --prompt-text-color: white;
+  --option-text-color: #5F6368;
```

Comment:
Do we want to let publishers override the text color?

Reply:
In our UX talks we considered it as an adjustable UI element - we could decide otherwise though